nautobot/docs/plugins/development.md

```diff
@@ -114,37 +146,43 @@ Nautobot looks for the `config` variable within a plugin's `__init__.py` to load
 
 The configurable attributes for a `PluginConfig` are listed below in alphabetical order.
 
-| Name | Description |
-| ---- | ----------- |
-| `author` | Name of plugin's author |
-| `author_email` | Author's public email address |
-| `base_url` | (Optional) Base path to use for plugin URLs. If not specified, the project's `name` will be used. |
-| `caching_config` | Plugin-specific cache configuration |
-| `custom_validators` | The dotted path to the list of custom validator classes (default: `custom_validators.custom_validators`) |
-| `datasource_contents` | The dotted path to the list of datasource (Git, etc.) content types to register (default: `datasources.datasource_contents`) |
-| `default_settings` | A dictionary of configuration parameters and their default values |
-| `description` | Brief description of the plugin's purpose |
-| `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.graphql_types)` |
-| `installed_apps` | A list of additional Django application dependencies to automatically enable when the plugin is activated (you must still make sure these underlying dependent libraries are installed) |
-| `jobs` | The dotted path to the list of Job classes (default: `jobs.jobs`) |
-| `max_version` | Maximum version of Nautobot with which the plugin is compatible |
-| `menu_items` | The dotted path to the list of menu items provided by the plugin (default: `navigation.menu_items`) |
-| `middleware` | A list of middleware classes to append after Nautobot's built-in middleware |
-| `min_version` | Minimum version of Nautobot with which the plugin is compatible |
-| `name` | Raw plugin name; same as the plugin's source directory |
-| `required_settings` | A list of any configuration parameters that **must** be defined by the user |
-| `template_extensions` | The dotted path to the list of template extension classes (default: `template_content.template_extensions`) |
-| `verbose_name` | Human-friendly name for the plugin |
-| `version` | Current release ([semantic versioning](https://semver.org/) is encouraged) |
+| Name                  | Description                                                                                                                                                                             |
```

Comment:
I don't see a big need to reformat this table - did you make any actual content changes here?

Reply:
No changes. It formats nicer to read, except on small screens. Reverting.